.changeset/itchy-donuts-love.md

```diff
@@ -0,0 +1,21 @@
+---
+'@astrojs/netlify': minor
+---
+
+Adds "includedFiles" and "excludedFiles" options. These allow extra files to be deployed in the SSR function bundle.
+
+When an Astro site using `server` or `hybrid` rendering is deployed to Netlify, the generated functions trace the server dependencies and include any that may be needed in SSR. However sometimes you may want to include extra files that are not detected as dependencies, such as files that are loaded using `fs` functions. Also, you may sometimes want to specifically exclude dependencies that are bundled automatically. For example, you may have a Node module that includes a large binary. The `"includedFiles"` and `"excludedFiles"` options allow you to do these. Both options support glob patterns, so you can include/exclude multiple files at once.
```

Comment:
```suggestion
When an Astro site using `server` or `hybrid` rendering is deployed to Netlify, the generated functions trace the server dependencies and include any that may be needed in SSR. However sometimes you may want to include extra files that are not detected as dependencies, such as files that are loaded using `fs` functions. Also, you may sometimes want to specifically exclude dependencies that are bundled automatically. For example, you may have a Node module that includes a large binary.

The `includedFiles` and `excludedFiles` options allow you specify these inclusions and exclusions as an array of file paths relative to the project root. Both options support glob patterns, so you can include/exclude multiple files at once.
```
Just thought this paragraph was getting a *little* long, so maybe breaking it up to separate where the use case is described vs the options themselves?